src/graphview/basicgraphview.h

```diff
@@ -90,11 +93,10 @@ protected slots:
     double getClosestPoint(double coordinate);
 
     QVector<QString> _tickLabels;
-    QList<QCPItemText *> _notesItems;
+    QList< std::shared_ptr<NoteItem> > _notesItems;
```

Comment:
Why std::shared_ptr and not Qt version?

Reply:
Forgotten that Qt has its own, reading [this forum post](https://forum.qt.io/topic/19073/any-difference-between-std-shared_ptr-and-qsharedpointer/4) I guess probably best to swap it for qt version

Reply:
Solved in 0162eb8